src/query/models/tag.go

```diff
@@ -145,17 +146,47 @@ func (m Matchers) ToTags() (Tags, error) {
 	return Normalize(tags), nil
 }
 
-// ID returns a string representation of the tags
-func (t Tags) ID() string {
-	b := make([]byte, 0, len(t))
+// StringID returns a string representation of the tags
+func (t Tags) StringID() string {
+	var (
+		idLen      = t.IDLen()
+		strBuilder = strings.Builder{}
```

Comment:
1.9 is over a year old at this point, 1.10 is over 6 months old. 1.11 just came out. 

Any reason you want to support 1.9?

Reply:
Probably ok to drop support, since we're only supporting built binaries really.  We would like most people to use all this software as docker images or as binaries built from the tagged releases.

Reply:
Nah, i agree that we should drop 1.9 support. Even prom only supports 1.10+ https://github.com/prometheus/prometheus/pull/3856. We should call it out in our README though.

Reply:
Ah yup, @richardartoul want to update the README.md (in a minimal way, we can update formatting/etc of it later) that we only support 1.10 at this time?